plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -4886,3 +4886,100 @@ func TestInterPodAffinityAnnotationsWithMultipleNodes(t *testing.T) {
 		}
 	}
 }
+
+func benchmarkPodMatchesNodeLabels(pods []*v1.Pod, nodes []*v1.Node, b *testing.B) {
```

Comment:
@k82cn - Sure, if you want something similar to below in scheduler_perf, the support is almost there.
```
(pprof) top100 cum
20ms of 16560ms total ( 0.12%)
      flat  flat%   sum%        cum   cum%
      20ms  0.12%  0.12%       20ms  0.12%  runtime.mallocgc
         0     0%  0.12%       20ms  0.12%  k8s.io/kubernetes/pkg/api/validation.AccumulateUniqueHostPorts
         0     0%  0.12%       20ms  0.12%  k8s.io/kubernetes/pkg/api/validation.ValidatePod
         0     0%  0.12%       20ms  0.12%  k8s.io/kubernetes/pkg/api/validation.ValidatePodSpec
         0     0%  0.12%       20ms  0.12%  k8s.io/kubernetes/pkg/api/validation.checkHostPortConflicts
```
We are working on integrating this into a prometheus docker container with expression browser but that is beyond 1.8 I guess.